### PR TITLE
Added and removed some of the tags

### PR DESCRIPTION
**Pull request by amadejpapez**
I went through the list of available tags and here are some of the changes I made:
- removed ```Pretty Good Privacy``` as it is the same as ```PGP```
- removed ```Cryptocurrency``` as it was only used for Bitcoin and there is already ```Cryptocurrency wallet``` tag
- removed some of the more broad tags as they were used only on some regexes and I think there are better more specific tags
- renamed ```Time``` to ```UNIX Timestamp```
- added ```CTF flag``` tag, instead of having them under ```Cyber Security```
- added ```IP``` tag for both IPv4 and IPv6
- added ```Finance``` tag to Laser Card Number as it was missing

### CODE REVIEW

**Comment by piatrashkakanstantinass** [comment]
Don't yoy think that tags like PGP Public key and PGP Private key do not make a lot of sense. These tags are just the names of entries. I would say that it is better to change these to PGP. This may be the same for some other tags aswell.

**Comment by amadejpapez** [comment]
Yeah agree. Do you see any other tags like this that could be removed?

**Comment by piatrashkakanstantinass** [comment]
IPv6, IPv4, MaestroCC, Mailgun. Also rename `CTF flag` to `CTF Flag`

**Review by piatrashkakanstantinass — APPROVED** [review]

File: pywhat/Data/regex.json [hunk 1]
```diff
@@ -238,7 +225,7 @@
       "Tags": [
          "Identifiers",
          "Networking",
-         "IPv6"
```
Comment (bee-san) [inline pywhat/Data/regex.json:241]: The reason we used specific tags here was so people could pick and choose what they wanted specifically, if we can filter on names this isn't needed :)

Comment (amadejpapez) [reply pywhat/Data/regex.json:241]: IPv4 and IPv6 are the ones that may actually come useful, so they can be added back yeah. Are there any others?

Comment (piatrashkakanstantinass) [reply pywhat/Data/regex.json:241]: In this case, it may be better to add individual tags to all entries.

**Pull request merged**